src/Extension/ExtensionInterface.php

```diff
@@ -31,35 +31,35 @@ interface ExtensionInterface
      *
      * @return TokenParserInterface[]
      */
-    public function getTokenParsers();
+    public function getTokenParsers(): array;
```

Comment:
Should we also update the PHPDoc return type to `array<TokenParserInterface>`, or keeping `TokenParserInterface[]` is fine?

Reply:
it is not more strict. It is strictly equivalent.

Reply:
Ah, I misread/misremembered https://phpstan.org/writing-php-code/phpdoc-types#general-arrays